sdk/servicebus/azure-messaging-servicebus-stress/src/main/java/com/azure/messaging/servicebus/stress/util/TestUtils.java

```diff
@@ -64,7 +66,7 @@ static ServiceBusClientBuilder.ServiceBusProcessorClientBuilder getProcessorBuil
             .disableAutoComplete();
     }
 
-    static ServiceBusClientBuilder.ServiceBusSessionProcessorClientBuilder getSessionProcessorBuilder(ScenarioOptions options) {
+    public static ServiceBusClientBuilder.ServiceBusSessionProcessorClientBuilder getSessionProcessorBuilder(ScenarioOptions options) {
```

Comment:
nit; later, when we stress/perf session processor, we should export the env var `com.azure.messaging.servicebus.session.processor.asyncReceive.v2` with vaule `true`  to opt-in v2 session.

Reply:
yep, I have better session tests in another branch with the flag set